### PR TITLE
chore: laydown initial repository layout

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This initializes the repository to create a UDS Package Valkey

## Related Issue

Fixes https://github.com/defenseunicorns/uds-package-gitlab/issues/97

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Other (security config, docs update, etc)

## Checklist before merging

- [X] Test, docs, adr added or updated as needed
- [X] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-package-valkey/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      "https://raw.githubusercontent.com/defenseunicorns/uds-cli/v0.10.4/tasks.schema.json": [
+        "tasks.yaml",
+        "tasks/**/*.yaml",
+        "src/**/validate.yaml"
+      ],
+      "https://raw.githubusercontent.com/defenseunicorns/zarf/v0.33.1/zarf.schema.json": [
+        "zarf.yaml"
+      ]
+    },
+  }
```
Comment (zachariahmiller) [inline .vscode/settings.json:16]:
```suggestion
  }
  
```
newline

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# This repository is owned by the Software Factory Team
+/* @defenseunicorns/swf
+
+# Additional privileged files
+/CODEOWNERS @jeff-mccoy @austenbryan
```
Comment (zachariahmiller) [inline CODEOWNERS:5]: need to add austen 😆

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (zachariahmiller) [inline LICENSE:189]: Fill?

File: release-please-config.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        "prerelease-type": "uds",
+        "extra-files": [
+          "bundle/uds-bundle.yaml",
+          "tasks/publish.yaml",
+          "zarf.yaml"
+        ]
+      }
+    },
+    "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json"
+  }
```
Comment (zachariahmiller) [inline release-please-config.json:24]:
```suggestion
  }
  
```

File: renovate.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      },
+      {
+        "groupName": "Valkey Package Dependencies",
+        "labels": ["package-deps"],
+        "commitMessageTopic": "package-deps",
+        "matchDatasources": ["docker", "helm", "git-tags"]
+      }
+    ]
+  }
+
```
Comment (zachariahmiller) [inline renovate.json:25]:
```suggestion

```
trailing spaces

File: tests/zarf.yaml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/defenseunicorns/zarf/main/zarf.schema.json
+kind: ZarfPackageConfig
+metadata:
+  name: valkey-test
```
Comment (zachariahmiller) [inline tests/zarf.yaml:4]: ❤️

**Review by zachariahmiller — CHANGES_REQUESTED** [review]
See comments for the couple minimal changes otherwise looks great!

File: CODEOWNERS (hunk 2 above)
Comment (Racer159) [reply CODEOWNERS:5]: yeah @oates how do you want to handle this?

File: LICENSE (hunk 3 above)
Comment (Racer159) [reply LICENSE:189]: PGO too lol

Comment (Racer159) [reply LICENSE:189]: Ahhhhhh they're all like that!

Comment (Racer159) [reply LICENSE]: filled and created an issue: https://github.com/defenseunicorns/uds-software-factory/issues/64

**Pull request merged**